projects/mercury/src/metadata/views/facets/DateSelectionFacet.js

```diff
@@ -41,26 +41,37 @@ const DateSelectionFacet = (props: MetadataViewFacetProperties) => {
         if (val.toString() === 'Invalid Date') {
             return false;
         }
-        return (val >= minDate && val <= maxDate);
+        // FNS converts two-digits (XX) input year to 1900+XX. Thus, only 4 digits year are considered as valid.
```

Comment:
FNS?

Reply:
mistakenly guessed as fns fomatter is used, updated the comment